tests/integration/test_vulnerability_detector/test_providers/data/test_cases/cases_update_interval.yaml

```diff
@@ -8,6 +8,27 @@
     provider_name: 'Amazon Linux 1'
     update_interval: '5s'
 
+- name: 'Amazon_Linux 2'
+  description: 'Test update interval 5s Amazon Linux'
```

Comment:
```suggestion
  description: 'Test update interval 5s Amazon Linux 2'
```

Reply:
Fixed in commit https://github.com/wazuh/wazuh-qa/pull/2911/commits/1b19247695cb3c80a4388c6ca67963e80f244b46

Reply:
fixed in commit https://github.com/wazuh/wazuh-qa/pull/2911/commits/1b19247695cb3c80a4388c6ca67963e80f244b46